modules/update-preparing-evaluate-alerts.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-prepare.adoc
+
+[id="update-preparing-evaluate-alerts_{context}"]
+= Reviewing alerts to identify uses of removed APIs
+
+{product-title} 4.8 introduced two new alerts that fire when an API that will be removed in the next release is in use:
```

Comment:
I find this sentence accurate but a bit hard to follow. Perhaps it could be more direct as something like, "...introduced two new alerts that tell you when you are using an API that will be removed in the next release" or even a small transposition, such as: "...when an API is in use that will be removed in the next release". Up to you if you find either of these helpful or not. The more I read it, though, maybe your original form does make the most sense. 🤷